foodplanner/lib/components/image.dart

```diff
@@ -22,4 +22,17 @@ class FoodImage extends StatelessWidget {
           }
         });
   }
+
+  Future<String?> loadImageAndToken() async {
+    String? jwtToken = await AuthProvider().retrieveToken();
+
+    var apiClient = ApiClient();
+    apiClient.addDefaultHeader('Image authorization', 'Bearer $jwtToken');
```

Comment:
Virker det med "Image authorization"?
```suggestion
    apiClient.addDefaultHeader('Authorization', 'Bearer $jwtToken');
```